.github/workflows/release.yml

```diff
@@ -174,7 +174,10 @@ jobs:
           pattern: artifacts-*
           path: target/distrib/
           merge-multiple: true
-
+      - name: Cargo Deb
```

Comment:
i don't get it. i thought we still haven't discussed if it is our responsibility to maintain packages

i am making a code suggestion here so that this will be a discussion so it can be minimized when resolved

ref #294

Reply:
Usually projects release the package for the various distro, in a different repo I think that is the case when the project will grrow.
Right now I can be the debian maintainer, as the package it is just the binarry generated from rust in a dedicated compressed package, is not something like for Nix.

Reply:
> in a different repo I think that is the case when the project will grrow

normally i would agree to move it later, but files in releases are something people will use in scripts, so its not very good to modify or remove them. so it has to be moved there right away.

why even have it in the compiler repo? i don't think there is a valid reason for having it here rather than in a separate one

Reply:
Debian packages are easy to use and not that hard to maintain. I see the reason to support just the most popular distributions so that we can maintain a good experience of the installation process.